packages/grid/_modules_/grid/components/panel/GridPreferencesPanel.tsx

```diff
@@ -22,9 +22,11 @@ export const GridPreferencesPanel = React.forwardRef<
   return (
     <rootProps.components.Panel
       ref={ref}
+      as={rootProps.components.BasePopper}
       open={columns.length > 0 && preferencePanelState.open}
       {...rootProps.componentsProps?.panel}
```

Comment:
If the `as` prop is applied inside `GridPanel`, then it doesn't need to be applied here again.